package.json

```diff
@@ -154,8 +154,8 @@
     "_private:proxy:build": "cargo build --all-features --all-targets",
     "_private:proxy:build:release": "cargo build --release",
     "_private:proxy:clean": "cargo clean",
-    "_private:proxy:start:test": "cargo build --bin git-remote-rad && cargo run -- --test",
-    "_private:proxy:start:test:watch": "cargo build --bin git-remote-rad && cargo watch -x 'run -- --test'",
+    "_private:proxy:start:test": "cargo build --release --bin git-remote-rad && cargo run --release -- --test",
```

Comment:
Won't run this with `--release` slow things down considerably?

Reply:
It speeds up signing things with the keystore

Reply:
Papoy!